internal/api/grpc/system/integration_test/instance_test.go

```diff
@@ -102,7 +102,7 @@ func TestServer_ListInstances(t *testing.T) {
 				require.Error(t, err)
 				return
 			}
-			require.NoError(t, err)
+			assert.NoError(t, err)
```

Comment:
When changing this from `require` to `assert`, the test does not get terminated in case of a unexepected error. As the length check below is also `assert`, the following `for` loop will throw an out of bounds panic on the `got` slice if there was an unexpected error.

There should be some kind of termination of the test before we reach the `for` loop. Either use `require` or a conditional `return`. Personaly I favor keeping `require` because it's cleaner.